frontend/src/static/css/style.scss

```diff
@@ -180,8 +179,10 @@ header{
             padding-right:0.25rem;
         }
 
-        &--tooltip{
+        &--tooltip, &--repo-url-button{
```

Comment:
`tooltip` should probably be changed to `codeview-button`

Reply:
Have changed